shimmingtoolbox/cli/b1shim.py

```diff
@@ -52,12 +53,15 @@ def b1shim_cli(fname_b1_map, fname_mask, fname_cp_weights, algorithm, target, fn
     nib.save(nii_b1, fname_nii_b1)
     file_json_b1 = open(os.path.join(path_output, 'TB1maps_uncombined.json'), mode='w')
     json.dump(json_b1, file_json_b1)
+    b1_map_combined = b1_map.sum(axis=-1)
+    nii_b1_map_combined = nib.Nifti1Image(b1_map_combined, nii_b1.affine, header=nii_b1.header)
 
     # Load static anatomical mask
     if fname_mask is not None:
         nii_mask = nib.load(fname_mask)
+        mask_resampled = resample_from_to(nii_mask, nii_b1_map_combined, order=1, mode='grid-constant').get_fdata()
```

Comment:
I would suggest having a figure or a debug option to look at the resampled mask.

Reply:
The mask is visible on the output figure
![image](https://user-images.githubusercontent.com/54723104/146063794-b857342d-7895-4148-9b30-476fed34fd20.png)